js/imageeditor/shared/core/editor.ts

```diff
@@ -1166,7 +1169,11 @@ export class ImageEditor {
 			width: container_box.width,
 			height: container_box.height,
 			backgroundAlpha: this.dark ? 0 : 1,
-			backgroundColor: this.dark ? 0x333333 : 0xffffff,
+			backgroundColor: this.dark
```

Comment:
Shouldn't this depend on the theme?

Reply:
Yes, we dont need this branch, will remove.